acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/das/DasSyncAcceptanceTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Consensys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.test.acceptance.das;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.IntStream;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
+import tech.pegasys.teku.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.teku.test.acceptance.dsl.TekuBeaconNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuNodeConfigBuilder;
+
+public class DasSyncAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void shouldSyncToNodeWithGreaterFinalizedEpoch() throws Exception {
+    final TekuBeaconNode primaryNode =
+        createTekuBeaconNode(
+            createConfigBuilder()
+                .withRealNetwork()
+                .withDasExtraCustodySubnetCount(128 - 4)
```

Comment:
it will cap it on 128 anyway
i'd replace this to 128 for readability

Reply:
Does it look better this way: 291cf5e ?